dask-values.yaml

```diff
@@ -0,0 +1,42 @@
+gateway:
+  proxyToken: <SECRET>
```

Comment:
When put in the right spot, both `<SECRET>` values will be in the `secret` folder, this was just to get something up for people to see.